packages/react-instantsearch/src/connectors/connectHierarchicalMenu.js

```diff
@@ -86,7 +86,7 @@ const sortBy = ['name:asc'];
  * @propType {string[]} [rootPath=null] - The already selected and hidden path.
  * @propType {boolean} [showParentLevel=true] - Flag to set if the parent level should be displayed.
  * @providedPropType {function} refine - a function to toggle a refinement
- * @providedPropType {function} createURL - a function to generate a URL for the corresponding state
+ * @providedPropType {function} createURL - a function to generate a URL for the corresponding searchState
```

Comment:
On this one I don't know. I renamed the state parameter of the createURL function, searchState on all examples. But we can go for the literal version "search state" :)